wgpu-core/src/validation.rs

```diff
@@ -1223,35 +1219,23 @@ impl Interface {
                         }
                     }
                 }
+                // TODO: front_facing, sample_index, and sample_mask builtin's should all increase
+                // components count for fragment input.
                 Varying::BuiltIn(_) => {}
```

Comment:
I noticed a few (here and 3 new comments below) potential inconsistencies with https://gpuweb.github.io/gpuweb/#abstract-opdef-validating-inter-stage-interfaces

Reply:
Lets lift this into an issue

Reply:
Opened issue https://github.com/gfx-rs/wgpu/issues/5577